conga-aem-definitions/src/main/templates/aem-dispatcher/author/vhost_author.ssl.conf.hbs

```diff
@@ -7,6 +7,11 @@
 # Virtual host for author (HTTPS)
 <VirtualHost {{httpd.serverListenAddressSsl}}:{{httpd.serverPortSsl}}>
 
+{{~#if httpd.http2.enabled}}
+# Enable HTTP/2
```

Comment:
i think this block should be moved to vhost_author.partials.hbs
and put further down in this file, e.g. after the block "serverAliasNames"

Reply:
Changed in latest commit